src/status_im/signing/core.cljs

```diff
@@ -535,7 +534,7 @@
                                  {:to    to-norm
                                   :value amount-hex}
                                  {:to   (ethereum/normalized-hex address)
-                                  :data (abi-spec/encode
+                                  :data (status/encodeFunctionCall
```

Comment:
just wondering, could it be something like `(status/encodeTransfer [to-norm amount-hex])` and go-ethereum with sol file would be used ?

Reply:
same question for decodeParameters

Reply:
sorry, i'm not sure I clearly understand your question :) 
- do u mean u don't like the name `encodeFunctionCall`, and want to replace it to `encodeTransfer`?  actually the names i refferenced is [here](https://web3js.readthedocs.io/en/v1.2.7/web3-eth-abi.html#encodefunctioncall)
- go-ethereum don't need to use sol file, u can c the corresponding PR on the side of `status-go` 
@flexsurfer